### PR TITLE
[MPS] Fix for matmul errors in test consistency

### PR DESCRIPTION
**Pull request by kulinseth**
* Handle empty input with non-empty output
* Remove transpose options from mm op

### CODE REVIEW

**Review by albanD — COMMENTED** [review]
Should the test be re-enabled?

**Review by malfet — CHANGES_REQUESTED** [review]
+1 on Alban's feedback: should this change include some testing?

**Comment by kulinseth** [comment]
> +1 on Alban's feedback: should this change include some testing?

Agreed, the testConsistency `nn.functional.linear` still has some issues. I will enable a subset which was fixed.

**Comment by kulinseth** [comment]
> +1 on Alban's feedback: should this change include some testing?

@malfet , can you please take a look now.

File: aten/src/ATen/native/mps/operations/Activation.mm [hunk 1]
```diff
@@ -417,6 +417,9 @@ Tensor relu_mps(const Tensor& self) {
   using CachedGraph = MPSUnaryCachedGraph;
   TORCH_CHECK(output.is_mps());
 
+  if(output.numel() == 0)
+    return;
+
```
Comment (malfet) [inline aten/src/ATen/native/mps/operations/Activation.mm:422]: nit: per PyTorch coding standard we should use curly braces even for single line return blocks
```suggestion
  if(output.numel() == 0) {
    return;
  }

```

File: aten/src/ATen/native/mps/operations/Linear.mm [hunk 2]
```diff
@@ -46,6 +46,9 @@ Tensor _mps_linear(
 
   TORCH_CHECK(output.is_mps());
 
+  if(output.numel() == 0)
+    return output;
+
```
Comment (malfet) [inline aten/src/ATen/native/mps/operations/Linear.mm:51]: nit
```suggestion
  if(output.numel() == 0) {
    return output;
  }

```

File: aten/src/ATen/native/mps/operations/PointwiseOps.mm [hunk 3]
```diff
@@ -18,6 +18,10 @@
   if (&output != &self) {
     output.resize_(output.sizes());
   }
+
+  if(output.numel() == 0)
+    return output;
+
```
Comment (malfet) [inline aten/src/ATen/native/mps/operations/PointwiseOps.mm:24]:
```suggestion
  if(output.numel() == 0) {
    return output;
  }

```

**Review by malfet — APPROVED** [review]
LGTM, see nits

**Comment by malfet** [comment]
Added `ciflow/mps` tag to schedule just the tests we care about (plan to add action that auto assigns this label soon)

**Comment by kulinseth** [comment]
> Added `ciflow/mps` tag to schedule just the tests we care about (plan to add action that auto assigns this label soon)

Sound great.
